net/unbound/Makefile

```diff
@@ -23,49 +23,101 @@ PKG_CPE_ID:=cpe:/a:nlnetlabs:unbound
 PKG_BUILD_PARALLEL:=1
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
 
 include $(INCLUDE_DIR)/package.mk
 
 define Package/unbound/Default
-  TITLE:=Validating Recursive DNS Server
-  URL:=https://www.unbound.net/
-  DEPENDS:=+libopenssl +@OPENSSL_WITH_EC
-endef
-
-define Package/unbound
-  $(call Package/unbound/Default)
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=IP Addresses and Names
   USERID:=unbound:unbound
-  TITLE+= (daemon)
+  TITLE:=Recursive DNS Server
+  URL:=https://nlnetlabs.nl/projects/unbound/about
+  DEPENDS:=+libopenssl +@OPENSSL_WITH_EC
+endef
+
+define Package/unbound-daemon
+  $(call Package/unbound/Default)
+  TITLE+= (daemon, light duty)
   DEPENDS+= +libunbound
+  VARIANT:=light
+endef
+
+define Package/unbound-daemon/description
+  This package contains the Unbound daemon with basic includes
+  necessary to meet the needs of UCI/LuCI configuration optoins.
 endef
 
-define Package/unbound/description
-  This package contains the Unbound daemon.
+define Package/unbound-daemon-heavy
+  $(call Package/unbound/Default)
+  TITLE+= (daemon, heavy duty)
+  DEPENDS+= +libunbound-heavy +libpthread +libevent2 +libevent2-pthreads
+  VARIANT:=heavy
```

Comment:
I would suggest using `full` as the name if the variant is geared more for complete features instead of "heavy" features.  We already have `dnsmasq`, `dnsmasq-dhcpv6`, `dnsmasq-full, `unbound-full` would seem more consistent.

Reply:
I understand your point but `full` is not a match for what this is. With dnsmas full is _full_ including its pseudo authoritative chunks of code. With Unbound here this is not the case. The names were specifically picked for "light duty" and "heavy duty" as noted in menuconfig suffix `(daemon, heavy duty)`. The changes directly relate to how much traffic Unbound can handle efficiently. Another level example I have not chosen to tackle this round is `-python`. Unbound has a plug-in module for this, but it's huge and so are the python minimum dependencies.

Reply:
So `heavy` is for handling heavy traffic, not for containing heavy features.  Got it.